content/blog/2017/03/2017-03-20-security-updates.adoc

```diff
@@ -13,7 +13,7 @@ Multiple Jenkins plugins received updates today that fix several security vulner
 * plugin:distfork[Distributed Fork]
 * plugin:email-ext[Email Extension (Email-ext)]
 * plugin:mailer[Mailer]
-* plugin:ssh-slaves[SSH Slaves]
+* plugin:ssh-slaves[SSH Agents]
```

Comment:
I think it's called the SSH Build Agents plugin now, not sure if you need the full name.

Reply:
```suggestion
* plugin:ssh-slaves[SSH Build Agents]
```